### ISSUE TITLE
Add option to enable more granular date/time info below VODs and Clips

### ISSUE THREAD

**Issue by Taizunz** [FrankerFaceZ/FrankerFaceZ#1033]
**Is your feature request related to a problem? Please describe.**
I've always been a bit frustrated by the lack of displaying full date/time info under videos/clips natively on Twitch (or at least surfacing it when hovering over the video thumbnail corner, which at the moment only surfaces the creation date, not the time)

**Describe the solution you'd like**
An option to show the full creation date/time of videos and clips below the player to replace the current "X days ago"-display, possibly with multiple levels of granularity to select from in the FFZ Control Center and a custom option to let the user format it to their liking based on the Day.js library, so it falls in line with existing date/time options we currently have available for other FFZ features.

Additionally this feature is also a request to have it added to both types of clips-pages (`clips.twitch.tv/<slug>` + `twitch.tv/<channel>/clip/<slug>`) to keep it consistent. At the moment there's no creation date info displayed at all on the `twitch.tv/<channel>/clip/<slug>` pages.

**Additional context**
I'm not a programmer so I have no clue how tedious this would be to add, but I do know how to look up information, and I noticed that for both videos and clips the "createdAt"-field is returned in one of the many GQL responses received when loading a video/clip page which has the full date and time of video/clip creation, so I assume this type of feature wouldn't require any subsequent API-hits beyond what is already requested upon a normal pageload.

A few examples of what I imagine this feature could look like for the end user, depending on their own date/time format settings applied in FFZ Control Center:
![2021-04-30_22-42-11](https://user-images.githubusercontent.com/15648000/116753324-383dcf80-aa07-11eb-8426-e28c04f9553b.png)
![2021-04-30_22-40-34](https://user-images.githubusercontent.com/15648000/116753326-38d66600-aa07-11eb-8bd3-0dfc88223875.png)
![chrome_2021-04-30_22-39-13](https://user-images.githubusercontent.com/15648000/116753328-38d66600-aa07-11eb-8cb4-6a619044b2d9.png)

**Comment by knakamura8** [comment]
Agreed.  This would be very helpful when viewing, for example, old VODs and clips which only provide information to the effective resolution of "years" e.g. "2 years ago."

**Comment by knakamura8** [comment]
Agreed. This would be very helpful when viewing, for example, old VODs and clips which only provide information to the effective resolution of "years," e.g. "2 years ago."
